components/skus/browser/resources/skus_internals.tsx

```diff
@@ -3,12 +3,55 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import * as SkusInternalsMojo from 'gen/brave/components/skus/common/skus_internals.mojom.m.js'
 import * as React from 'react'
+import styled from 'styled-components'
 import { render } from 'react-dom'
+import { JsonView } from 'react-json-view-lite';
+import 'react-json-view-lite/dist/index.css';
+
+const Container = styled.div`
+  margin: 4px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+`
+
+const API = SkusInternalsMojo.SkusInternals.getRemote()
 
 function App() {
+  const [skusState, setSkusState] = React.useState('{}')
+
+  const resetSkusState = () => {
+    API.resetSkusState()
+    setSkusState('{}')
+  }
+
+  const getSkusState = () => {
+    API.getSkusState().then((r: any) => {
+      setSkusState(r.response)
+    })
+  }
+
   return (
-    <div>Log here</div>
+    <Container>
+      <h2>Skus internals</h2>
+      <ButtonContainer>
+        <input type="button" value="Reset SKUs state" onClick={() => {
+            if (window.confirm('Are you sure you want to reset SKUs state? This cannot be undone')) {
+              resetSkusState()
+            }
+          }} />
+        <input type="button" value="Fetch SKUs state" onClick={() => { getSkusState() }} />
+      </ButtonContainer>
+      <JsonView data={JSON.parse(skusState)} shouldInitiallyExpand={(level) => true} />
```

Comment:
FWIW, this will reparse `skusState` on every render, which might be often. Probably fine though because this is a debug page.

Reply:
Changed to parse only once when get data. 👍🏼